veil-macros/src/enums.rs

```diff
@@ -168,7 +168,7 @@ pub(super) fn derive_redact(
     for (variant, flags) in e.variants.iter().zip(variant_flags.into_iter()) {
         // Variant name redacting
         let variant_name = variant.ident.to_string();
-        let variant_name = if let Some(flags) = &flags.variant_flags {
+        let variant_name = if let Some(flags @ FieldFlags { skip: false, .. }) = &flags.variant_flags {
```

Comment:
This is for me: wth is `@`?

Reply:
it binds the flags variable if the pattern on the right matches
https://doc.rust-lang.org/stable/rust-by-example/flow_control/match/binding.html

Reply:
such an awesome feature :)